### DISCUSSION TITLE
Feature: Backup

### DISCUSSION THREAD

**Discussion by coolapso** [StreamController/StreamController#38] [Ideas]
Would be great to have a way to export / backup StreamController settings and eventually version control this settings.

**Comment by TannerGabriel** [comment]
Is there any update on this feature?

**Reply by coolapso** [reply]
Yes .. if I recall correctly this is implemented.

**Reply by TannerGabriel** [reply]
Thanks, I just found it.

**Comment by ImDevinC** [comment]
To clarify, in the page editor, you can export all pages.
